processor/k8sattributesprocessor/processor.go

```diff
@@ -108,6 +108,7 @@ func (kp *kubernetesprocessor) processLogs(ctx context.Context, ld plog.Logs) (p
 // processResource adds Pod metadata tags to resource based on pod association configuration
 func (kp *kubernetesprocessor) processResource(ctx context.Context, resource pcommon.Resource) {
 	podIdentifierKey, podIdentifierValue := extractPodID(ctx, resource.Attributes(), kp.podAssociations)
+	kp.logger.Debug("Evaluating Pod Identifier", zap.String("pod identifier key", podIdentifierKey), zap.Any("pod identifier value", podIdentifierValue))
```

Comment:
A nit, we are not typically using capital letters when emitting logs ([example](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/k8sattributesprocessor/internal/kube/client.go#L143))
Also, "pod identifier key"/"pod identifier value" seem a little bit redundant (considering the context maybe just "key"/"value" are enough) but it's subjective, so feel free to leave it as is or change